v1.2.yml

```diff
@@ -1263,28 +1310,36 @@ definitions:
         x-nullable: true
       student:
         type: string
+        x-validation: true
       type:
         type: string
-        x-nullable: true
 
   Teacher:
     type: object
     properties:
       id:
         type: string
+        x-validation: true
       created:
         type: string
         format: datetime
+        x-validation: true
+      credentials:
+        $ref: "#/definitions/Credentials"
         x-nullable: true
       district:
         type: string
+        x-validation: true
       email:
         type: string
         x-nullable: true
+      google_apps:
+        $ref: "#/definitions/GoogleApps"
+        x-nullable: true
       last_modified:
         type: string
         format: datetime
-        x-nullable: true
+        x-validation: true
       name:
         $ref: "#/definitions/Name"
       school:
```

Comment:
x-validation: true for school
sis_id can't be null
schools array should be x-validation: true as well